services/motion/builtin/move_request.go

```diff
@@ -61,8 +60,7 @@ const (
 type moveRequest struct {
 	requestType requestType
 	// geoPoseOrigin is only set if requestType == requestTypeMoveOnGlobe
-	geoPoseOrigin spatialmath.GeoPose
-	// poseOrigin is only set if requestType == requestTypeMoveOnMap
```

Comment:
Shouldn't this comment still be applicable?

Reply:
No.  PoseOrigin is used in all paths now